### DISCUSSION TITLE
Custom Application URL Protocol in Custom Links

### DISCUSSION THREAD

**Discussion by HMohammad2520** [netbox-community/netbox#18082] [Help Wanted!]
I am using a custum application on my workspace. 
I have a custom url protocol for it: myapp

When i use the url of my app as the link url in custom links, it returns an empty url.
> custom link url: myapp://<mydevice.primery_ip4.address>

How to add my url protocol to valid protocols of netbox as the custom link ?

**Comment by candlerb** [comment]
https://netboxlabs.com/docs/netbox/en/stable/configuration/security/#allowed_url_schemes

**Reply by HMohammad2520** [reply]
Thank you 
it worked ❤️ 

I think we should mention this in the Custom Links Documentation
https://netboxlabs.com/docs/netbox/en/stable/customization/custom-links/
